pkg/kubelet/prober/prober.go

```diff
@@ -297,7 +297,9 @@ func (eic *execInContainer) Stop() {
 func (eic *execInContainer) Start() error {
 	data, err := eic.run()
 	if eic.writer != nil {
-		eic.writer.Write(data)
+		if p, err := eic.writer.Write(data); err != nil {
+			klog.Errorf("Should write %d bytes but only write %d bytes: %v", len(data), p, err)
```

Comment:
Please use `ErrorS`.

```suggestion
			klog.ErrorS(err, "Unable to write all bytes from execInContainer", "expectedBytes", len(data), "actualBytes", p)
```

Reply:
ok, done.